src/util/psbt/mod.rs

```diff
@@ -612,6 +615,7 @@ mod tests {
                 },
                 version: 0,
                 xpub: Default::default(),
+                proprietary: Default::default(),
```

Comment:
nit: You have mixed usage of `Default::default()` and `BTreeMap::new()` in this file.